movement/move_accessor.py

```diff
@@ -42,7 +42,7 @@ class PosesAccessor:
     See [1]_ for more details.
 
     Methods/properties that are specific to this class can be used via
-    the `.poses` accessor, e.g. `ds.poses.to_dlc_df()`.
+    the `.move` accessor, e.g. `ds.move.to_dlc_df()`.
```

Comment:
This is my own fault, but all the monospace words in this docstring should be specified via double backtick \``.move`` (maybe good chance to do it now)?

Reply:
There seems to be a lot more occurrences in other files. I'll only change these in the accessor to keep it relevant. We can create another issue for fixing the rest.